frontend/src/assets/datasuggest.js

```diff
@@ -14,53 +14,62 @@ export default [
     type: "vibe",
     picture: imgSad,
     alt: "smiley sad",
+    query: "with_genres=35&with_genres=28& with_genres=16",
   },
   {
     key: "happy",
     type: "vibe",
     picture: imgHappy,
     alt: "smiley happy",
+    query: "with_genres=10749&with_genres=16& with_genres=35",
   },
   {
     key: "thinking",
     type: "vibe",
     picture: imgThinking,
     alt: "smiley thinking",
+    query: "with_genres=99&with_genres=36& with_genres=10752",
   },
   {
     key: "angry",
     type: "vibe",
     picture: imgAngry,
     alt: "smiley angry",
+    query: "with_genres=80&with_genres=27& with_genres=9648&with_genres=53",
   },
   {
     key: "old",
     type: "dateRelease",
     picture: imgOld,
     alt: "old",
+    query: "release_date.lte=1990-01-01",
   },
   {
     key: "young",
     type: "dateRelease",
     picture: imgYoung,
     alt: "young",
+    query: "release_date.lte=2010-01-01&release_date.gte=1990-01-01",
   },
   {
     key: "baby",
     type: "dateRelease",
     picture: imgBaby,
     alt: "baby",
+    query: "release_date.gte=2010-01-01&release_date=2010-01-01",
```

Comment:
"Send me movies more recent than 2010-01-01 *and* which went live on 2010-01-01" ? Have you tested it ?